azure-proto-compute/VirtualMachineContainer.cs

```diff
@@ -101,21 +103,21 @@ public async override Task<ArmOperation<VirtualMachine>> StartCreateAsync(string
         /// <summary>
         /// Construct an object used to create a VirtualMachine.
         /// </summary>
-        /// <param name="vmName"> The name of the Virtual Machine. </param>
+        /// <param name="hostName"> The hostname for the virtual machine. </param>
         /// <param name="adminUser"> The admin username to use. </param>
         /// <param name="adminPw"> The admin password to use. </param>
         /// <param name="nicId"> The network interface id to use. </param>
         /// <param name="aset"> The availability set to use. </param>
         /// <param name="location"> The location to create the Virtual Machine. </param>
         /// <returns> Object used to create a <see cref="VirtualMachine"/>. </returns>
-        public VirtualMachineModelBuilder Construct(string vmName, string adminUser, string adminPw, ResourceIdentifier nicId, AvailabilitySetData aset, LocationData location = null)
+        public VirtualMachineModelBuilder Construct(string hostName, string adminUser, string adminPw, ResourceIdentifier nicId, AvailabilitySetData aset, LocationData location = null)
```

Comment:
we should not using abbreviations on parameter names. Also would like to see we use ResourceIdentifier for AvalabilitySet as well.